packages/core/test/JWeditor.test.ts

```diff
@@ -36,6 +36,76 @@ describe('core', () => {
                     );
                     expect(expectedCommands).to.eql(['command-all', 'command-pc']);
                 });
+                it('should transform ctrl to CMD if no platform on mac', () => {
+                    const editor = new JWEditor();
+                    editor._platform = 'mac';
+                    editor.start();
+                    // todo: to remove when the corePlugin will be removed from editor.start().
+                    editor.eventManager.keymap._shortcutItems = [];
```

Comment:
Well actually, all the shortcutable commands from `CorePlugin` are not in `CorePlugin` anymore, so you should be able to remove this after you rebase on master.

Reply:
Some are still there